src/inference/src/dev/core_impl.cpp

```diff
@@ -592,6 +592,15 @@ ov::Plugin ov::CoreImpl::get_plugin(const std::string& pluginName) const {
             so = ov::util::load_shared_object(desc.libraryLocation.c_str());
             std::shared_ptr<ov::IPlugin> plugin_impl;
             reinterpret_cast<ov::CreatePluginFunc*>(ov::util::get_symbol(so, ov::create_plugin_function))(plugin_impl);
+            auto plugin_name = plugin_impl->get_device_name();
```

Comment:
```suggestion
            const auto& plugin_name = plugin_impl->get_device_name();
```
Apply if there will be any change to not restart CI. But I think Coverity will complain about make a copy here.